cocotb_build_libs.py

```diff
@@ -150,13 +161,32 @@ def _get_lib_ext_name():
 
 
 class build_ext(_build_ext):
+    def _uses_msvc(self):
+        if self.compiler == "msvc":
+            return True
+        if self.compiler is None:
+            return get_default_compiler() == "msvc"
+        else:
+            return "msvc" == getattr(self.compiler, "compiler_type", None)
+
     def run(self):
+        create_sxs_appconfig(self.get_ext_fullpath(os.path.join("cocotb", "simulator")))
```

Comment:
I noticed it was creating a file on Linux.
```suggestion
        if os.name == "nt":
            create_sxs_appconfig(self.get_ext_fullpath(os.path.join("cocotb", "simulator")))
```